services/time/README.md

```diff
@@ -1,10 +1,7 @@
 # exonum-time
 
-[![Travis Build Status](https://img.shields.io/travis/exonum/exonum/master.svg?label=Linux%20Build)](https://travis-ci.org/exonum/exonum)
```

Comment:
The idea behind all those badges is simple: they're visible on crates.io page and provide some basic info about crate (that it's tested, have a copy-left license and requires specific Rust version). At the same time, crates.io badge is pretty useless (the last version is already visible on docs.rs badge and it's the only thing you can find on crates.io and not in readme).

Reply:
So I intend to disagree with this change..